.github/workflows/requirements-locks.yml

```diff
@@ -31,5 +31,11 @@ jobs:
       - name: Generate riot locks
         run: scripts/compile-and-prune-test-requirements
 
-      - name: Check git diff
+      - name: Check locks diff
         run: scripts/check-diff '.riot/requirements/' 'Mismatches found between .riot/requirements/*.txt and riotfile.py. Run scripts/compile-and-prune-test-requirements and commit the result.'
+
+      - name: Generate min_compatible_versions
+        run: python scripts/min_compatible_versions.py
+
+      - name: Check min_compatible_versions diff
```

Comment:
do we want to check diff between `min_compatible_versions.csv` and `lib-injection/sources/min_compatible_versions.csv` as well?